code/modules/surgery/organs/helpers.dm

```diff
@@ -29,20 +29,14 @@
 /mob/living/carbon/getorgan(typepath)
 	return (locate(typepath) in internal_organs + external_organs)
 
-
-/mob/living/carbon/getorganszone(zone, subzones = 0, external = FALSE)
-	var/list/returnorg = list()
-	if(subzones)
-		// Include subzones - groin for chest, eyes and mouth for head
-		if(zone == BODY_ZONE_HEAD)
-			returnorg = getorganszone(BODY_ZONE_PRECISE_EYES) + getorganszone(BODY_ZONE_PRECISE_MOUTH)
-		if(zone == BODY_ZONE_CHEST)
-			returnorg = getorganszone(BODY_ZONE_PRECISE_GROIN)
-
+/mob/living/carbon/getorganszone(zone, include_children = 0)
+	. = list()
 	for(var/obj/item/organ/organ as anything in internal_organs + external_organs)
 		if(zone == organ.zone)
-			returnorg += organ
-	return returnorg
+			. += organ
+		else if(include_children && zone == deprecise_zone(organ.zone))
+			. += organ
+	return .
```

Comment:
The usage of `.` here degrades readability and I don't think this code is hot enough to warrant a micro-optimization like that.

Reply:
https://github.com/tgstation/tgstation/blob/master/.github/guides/STANDARDS.md#dot-variable-

Reply:
as in, `.` should not be being used for this